core/app/models/spree/shipping_method_stock_location.rb

```diff
@@ -0,0 +1,6 @@
+class ShippingMethodStockLocation
+  self.table_name = :spree_shipping_methods_stock_locations
```

Comment:
@athal7 pros and cons came to mind. I would start the rename here and in the rail engine add:

```ruby
Spree = Solidus
```

This way `Spree` starts to become a second class citizen and its `Solidus`.